lib/app-infos-cache.js

```diff
@@ -98,6 +98,12 @@ export class AppInfosCache {
     let lastError;
     try {
       await zip.readEntries(ipaPath, async ({entry, extractEntryTo}) => {
+        // For furutre reference:
+        // If the directory included `.app` (case insensitive) name as part of directory name
```

Comment:
If the directory name includes the '.app' suffix (case insensitive) like ..., then 'entry.fileName' would return 'Payload/something.App/filename'. The behavior is specific for unzip. Technically, such naming is possible, although plist info retrieval would fail.